test/integration/verify/controls/azurerm_subnet.rb

```diff
@@ -0,0 +1,25 @@
+resource_group  = attribute('resource_group',                default: nil)
+name            = attribute('subnet_name',                   default: nil)
+id              = attribute('subnet_id',                     default: nil)
+vnet            = attribute('vnet_name',                     default: nil)
+address_prefix  = attribute('subnet_address_prefix',         default: nil)
+nsg             = attribute('subnet_nsg',                    default: nil)
```

Comment:
Can you remove the extra white space here. We can align one space after `'subnet_address_prefix', `.